src/applications/disability-benefits/686c-674-v2/config/chapters/veteran-information/veteran-information/VeteranInformationComponent.jsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { isValid, format } from 'date-fns';
+
+import { CONTACTS } from '@department-of-veterans-affairs/component-library/contacts';
+
+import { genderLabels } from '~/platform/static-data/labels';
+import { selectProfile } from '~/platform/user/selectors';
+
+import { srSubstitute } from '~/platform/forms-system/src/js/utilities/ui/mask-string';
+
+import {
+  FORMAT_YMD_DATE_FNS,
+  FORMAT_READABLE_DATE_FNS,
+} from '../../../constants';
+
+import { parseDateToDateObj } from '../../../utilities';
+
+// separate each number so the screenreader reads "number ending with 1 2 3 4"
+// instead of "number ending with 1,234"
+const mask = value => {
+  const number = (value || '').toString().slice(-4);
+  return srSubstitute(
+    `●●●–●●–${number}`,
```

Comment:
I am unfamiliar with these special characters, can you confirm they are correct here?

Reply:
Wondering if we can use the html char codes? 
https://www.compart.com/en/unicode/U+25CF was the doc I found for the symbol we are currently using. Might ensure the code looks right as well in case someone's editor doesn't support these unicode chars

Reply:
This is the same component and set up as the example in `src/applications/appeals/10182` - Verified the setup with @Mottie